{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -73,6 +73,10 @@
 # https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#settings
 AWS_STORAGE_BUCKET_NAME = env('DJANGO_AWS_STORAGE_BUCKET_NAME')
 # https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#settings
+AWS_DEFAULT_ACL = env('DJANGO_AWS_DEFAULT_ACL', default=None)
+# https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#settings
+AWS_BUCKET_ACL = env('DJANGO_AWS_BUCKET_ACL', default=None)
```

Comment:
It says in their doc:

> Only used if `AWS_AUTO_CREATE_BUCKET=True`. The ACL of the created bucket.

Since that other setting is False by default and we don't override it, should we set this value? If someone changes the auto-create bucket setting after generating a project, isn't it up to them to make sure this is changed accordingly?

Reply:
I agree with @browniebroke: I specifically set `AWS_AUTO_CREATE_BUCKET` to `False` as otherwise that would yield unexpected result (maybe the end user wants to create a bucket himself, attaching a few policies to it prior to uploading anything there etc).

Reply:
> I specifically set AWS_AUTO_CREATE_BUCKET to False
@webyneter Where did you set this? I couldn't find it..